plugins/woocommerce/client/legacy/js/frontend/address-autocomplete.js

```diff
@@ -183,6 +191,7 @@ window.wc.addressAutocomplete.registerAddressAutocompleteProvider =
 		addressTypes.forEach( ( type ) => {
 			cacheAddressFields( type );
```

Comment:
Is it worth checking if `addressInputs[ type ]` already exists in the cache before updating it to avoid unnecessary DOM queries? Or are we intentionally overwriting the cached references?

Reply:
Thanks! Yeah, it makes sense to me. I figured as much but I thought I'd raise it here just incase for clarity. Fine to mark as resolved.